tools/example-templates/additional-files/A.cs

```diff
@@ -0,0 +1,19 @@
+class A
```

Comment:
I suspect we'll want to change this to be more descriptive at some point, but it's fine for the moment.